service/src/java/org/apache/hive/service/cli/CLIService.java

```diff
@@ -133,6 +135,13 @@ private void applyAuthorizationConfigPolicy(HiveConf newHiveConf) throws HiveExc
     ss.setIsHiveServerQuery(true);
     SessionState.start(ss);
     ss.applyAuthorizationPolicy();
```

Comment:
I slightly feel this `ss` can be immediately closed because this might be a hack to update `newHiveConf`